tests/unit/cn/kcompact_test.c

```diff
@@ -623,12 +623,6 @@ MTF_DEFINE_UTEST_PRE(kcompact_test, kcompact_fail, pre)
     if (run_kcompact(lcl_ti, 123))
         return;
     mapi_inject_unset(api);
-
-    api = mapi_idx_kvset_iter_next_key;
```

Comment:
Removed because element sources have no way of bubbling errors for better or for worse.